docker/release/centos7/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM centos:latest
+
+LABEL maintainer="Jumping Yang <yzpcreate@qq.com>"
+LABEL readme.md="https://github.com/Microsoft/omi/blob/master/docker/README.md"
+LABEL description="This Dockerfile will install the latest release of OMI."
+
+ARG rootpassword=newpassword
+
+SHELL ["/bin/bash", "-c"] 
+RUN echo $rootpassword |passwd root --stdin
+RUN yum update -y; \
+    yum install -y rpm-devel pam-devel openssl which rpm-build krb5-devel redhat-lsb-core;
+RUN rpm -Uvh http://packages.microsoft.com/config/rhel/7/packages-microsoft-prod.rpm
+RUN yum install -y omi
+RUN sed -i 's/httpsport=0/httpsport=0,5986/g' /etc/opt/omi/conf/omiserver.conf
```

Comment:
Technically, you only need 5986, I'd think. 0 is an artifact of `omiconfigeditor`.

How come you didn't use `omiconfigeditor` for this, much like the installation scripts do (for SCXcore) to expose 1270? Wouldn't that be a better more consistent way, and less sensitive to any changes we may make to the `omiserver.conf` file?